.circleci/config.yml

```diff
@@ -131,10 +131,45 @@ jobs:
             - /home/circleci/conda
           key: conda-deps-v1-{{ checksum "python/environment.yml" }}
 
+  spark-3-tests:
+    <<: *setup_base
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - conda-deps-v1-{{ checksum "python/environment.yml" }}
+      - *install_deps
+      - run:
+          name: Change Spark version
+          environment:
+          command: |
+            echo "3.0.0-SNAPSHOT" > spark-version.txt
```

Comment:
Why read this from a text file? I think it's more standard to read from an environment variable or a system property. Then there's less indirection.

Reply:
Sure, I made the change.